src/emcpy/plots/create_plots.py

```diff
@@ -1,14 +1,150 @@
 import numpy as np
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
 import matplotlib.pyplot as plt
+from matplotlib import rcParams
 from matplotlib.colors import Normalize
 from scipy.interpolate import interpn
+from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
+from emcpy.plots.map_tools import Domain, MapProjection
 from emcpy.stats import get_linear_regression
 
+__all__ = ['EMCPyPlots', 'CreatePlot', 'CreateMap']
 
-__all__ = ['CreatePlot']
 
+class EMCPyPlots:
 
-class CreatePlot:
+    def __init__(self):
+        return
+
+    def add_title(self, label,
+                  loc='center',
+                  fontsize=12,
+                  fontweight='normal',
+                  color='k',
+                  va='baseline'):
```

Comment:
can we keep this as `verticalalignment`? `va` means nothing to me besides Virginia or 'go' in Spanish